src/excmds.ts

```diff
@@ -4345,35 +4347,231 @@ const KILL_STACK: Element[] = []
+                                break
+                            case "t":
+                                newOpenMode = OpenMode.Tab
+                                break
+                            case "b":
+                                newOpenMode = OpenMode.BackgroundTab
+                                break
+                            case "w":
+                                newOpenMode = OpenMode.Window
+                                break
+                            case "wp":
+                                newOpenMode = OpenMode.WindowPrivate
+                                break
+                            case "h":
+                                newOpenMode = OpenMode.Highlight
+                                break
+                            case "i":
+                                newOpenMode = OpenMode.Images
+                                break
+                            case "I":
+                                newOpenMode = OpenMode.ImagesTab
+                                break
+                            case "k":
+                                newOpenMode = OpenMode.Kill
+                                break
+                            case "K":
+                                newOpenMode = OpenMode.KillTridactyl
+                                break
+                            case "z":
+                                newOpenMode = OpenMode.Scroll
+                                break
+                            case "s":
+                                newOpenMode = OpenMode.SaveResource
+                                break
+                            case "S":
+                                newOpenMode = OpenMode.SaveImage
+                                break
+                            case "a":
+                                newOpenMode = OpenMode.SaveAsResource
+                                break
+                            case "A":
+                                newOpenMode = OpenMode.SaveAsImage
+                                break
+                            case ";":
+                                newOpenMode = OpenMode.Semicolon
+                                break
+                            case "r":
+                                newOpenMode = OpenMode.TTSRead
+                                break
+                            case "P":
+                                newOpenMode = OpenMode.YankAlt
+                                break
+                            case "#":
+                                newOpenMode = OpenMode.YankAnchor
+                                break
+                            case "y":
+                                newOpenMode = OpenMode.YankLink
+                                break
+                            case "p":
+                                newOpenMode = OpenMode.YankText
+                                break
+                            default:
```

Comment:
`case "c":` will be so common that we would probably add dedicated (no-op) handling for it

Reply:
See above